src/backend/marsha/core/tests/test_xapi.py

```diff
@@ -0,0 +1,118 @@
+"""Tests for the xapi module of the Marsha project."""
```

Comment:
In your commit message:
"to ensure what is doing" -> "to make sure it is working as expected"